src/python-frontend/python_converter.cpp

```diff
@@ -1307,7 +1399,17 @@ void python_converter::get_var_assign(
   if (has_value && rhs != exprt("_init_undefined"))
   {
     if (lhs_symbol)
+    {
+      if (lhs_type == "str")
```

Comment:
Is this "str" hardcoded?

Reply:
Yes. **str** is a built-in class in Python used to represent strings: https://docs.python.org/3/library/stdtypes.html#str